File: DotNet/CesiumLanguageWriter/Advanced/ICesiumUnitSphericalListValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of unit spherical values.
    /// </summary>
    public interface ICesiumUnitSphericalListValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of unit spherical values.
        /// </summary>
        /// <param name="values">The values.</param>
        void WriteUnitSpherical(IEnumerable<UnitSpherical> values);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumUnitSphericalListValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a list of unit spherical values.
 */
@SuppressWarnings({
    "unused",
    "deprecation",
    "serial"
})
public interface ICesiumUnitSphericalListValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a list of unit spherical values.
    * @param values The values.
    */
    void writeUnitSpherical(Iterable<UnitSpherical> values);
}